demo/web/firebase_element/app_element.html

```diff
@@ -0,0 +1,39 @@
+<dom-module id="app-element">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <div class="horizontal center-justified layout">
+      <div class="vertical-section">
+        <h3>Dinosaurs by Height</h3>
+        <template is="dom-bind">
+          <firebase-collection
+              order-by-child="height"
+              location="https://dinosaur-facts.firebaseio.com/dinosaurs"
+              data="{{dinosaurs}}"></firebase-collection>
+          <template is="dom-repeat" items="[[dinosaurs]]" as="dinosaur">
```

Comment:
Seems to work, a list of dinosaurs is shown but I created `<app-element>` so I can avoid `<dom-bind>` but I missed this one.

The intention for `<app-element>` also was to be able to additionally integrate all demos in one single application page with a menu to switch between all the demos. 
Polymer.js uses `<iframe>` for this but I don't think this is the optimal solution for Dart.
I just run into an issue with this approach because https://github.com/dart-lang/polymer_elements/blob/master/lib/src/paper-styles/demo-pages.html doesn't work this way. It looks like it needs to be changed from `<style is="custom-style">` to `<dom-element id="some-id">` 
What do you think about how to proceed?